services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersValidateSimpleFlowSpec.groovy

```diff
@@ -0,0 +1,517 @@
+package org.openkilda.functionaltests.spec.switches
+
+import static org.junit.Assume.assumeTrue
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.PathHelper
+import org.openkilda.functionaltests.helpers.SwitchHelper
+import org.openkilda.messaging.command.switches.DeleteRulesAction
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+
+import groovy.transform.Memoized
+import org.springframework.beans.factory.annotation.Autowired
+import spock.lang.Narrative
+
+@Narrative("""This test suite check the meter validate request on a flow that contains 2+ switches.
+description of fields:
+- missing - those meters/rules, which is NOT exist on a switch, but it is exist in db
+- misconfigured - those meters which have different value (on a switch and in db) for the same parameter
+- excess - those meters/rules, which is exist on a switch, but it is NOT exist in db
+- proper - meters/rules value is the same on a switch and in db
+""")
+class MetersValidateSimpleFlowSpec extends BaseSpecification {
+    @Autowired
+    SwitchHelper switchHelper
+
+    def "Able to get meter validate information"() {
+        when: "Create a flow"
+        def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
+        def defaultMetersSrcSwitch = northbound.getAllMeters(srcSwitch.dpId)
+        def defaultMetersDstSwitch = northbound.getAllMeters(dstSwitch.dpId)
+        def flow = flowHelper.addFlow(flowHelper.randomFlow(srcSwitch, dstSwitch))
+
+        then: "One meter is automatically created on the both switches"
```

Comment:
```
when: "Create a flow"
then: "One meter is automatically created on the both switches"
```
^Not clear what is 'both switches' form this context. I think it's better to say 'on both src and dst switches'

Reply:
Fixed